lib/hanami/cli/generators/gem/app/package.json.erb

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "<%= underscored_app_name %>",
+  "private": true,
+  "scripts": {
+    "assets": "node config/assets.mjs"
+  },
+  "devDependencies": {
```

Comment:
```suggestion
  "dependencies": {
```

@timriley We need the NPM package to be installed in production as well for deployment reasons.

Reply:
Ohhh, great pickup, thank you! Fixed now.